### PR TITLE
Update package.json

### PR DESCRIPTION
**Pull request by VinodhiniV**
Updating lodash to ^4.17.15

### CODE REVIEW

**Comment by VinodhiniV** [comment]
We are heavily dependent on this module. Please update the lodash vuln asap

**Comment by jsdevel** [comment]
what issue are you having with the current version of lodash?

**Comment by VinodhiniV** [comment]
Prototype pollution in lodash versions <=4.17.11

**Comment by jsdevel** [comment]
Please update the commit message to reflect that you're upgrading lodash for this purpose, and squash your commits into one.

**Comment by VinodhiniV** [comment]
Hi @jsdevel , 

I had updated the commit message to reflect the purpose of lodash update and squashed the commit . 

Thanks!

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "soap",
-  "version": "0.28.0",
```
Comment (jsdevel) [inline package.json]: Please revert this.  We bump the version when we publish.

Comment (VinodhiniV) [reply package.json]: Sure. I have reverted the soap version change

**Comment by jsdevel** [comment]
Thanks!

**Pull request merged**